CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 
 ## Unreleased
 
+### Fixed
+
+- Roleselection for a lot of roles now considers all possible subroles one after another
```

Comment:
maybe make sure to mention that the previous issue was that traitor subroles were always assigned first, resulting in only having T-Subroles when the subrole limit was smaller or equal to the amount of traitor subroles installed